commit 90b77efcecb262823fadb67b0ce218846cd9e756
Author: LN Liberda <lauren@selfisekai.rocks>
Date:   Fri Feb 20 17:06:00 2026 -0800

    Remove now-obsolete/invalid patch on bytemuck-v1
    
    The patch was introduced to work with nightly Rust
    <https://crbug.com/480176523>. Then an upstream fix was rolled in
    I498189a701ed017cb9d07d958d296a6988963eb7, making this patch obsolete,
    however it still seems to apply on top of upstream, so it wasn't
    removed.
    
    Bug: 480176523
    Change-Id: I3466cd2423b658aa34b056eabc0b908f46f0a95c
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/7594600
    Commit-Queue: Nico Weber <thakis@chromium.org>
    Reviewed-by: Nico Weber <thakis@chromium.org>
    Reviewed-by: Łukasz Anforowicz <lukasza@chromium.org>
    Commit-Queue: Łukasz Anforowicz <lukasza@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1588192}

diff --git a/third_party/rust/chromium_crates_io/patches/bytemuck-v1/0101-Ad-hoc-bytemuck-fix-for-https-crbug.com-480176523.patch b/third_party/rust/chromium_crates_io/patches/bytemuck-v1/0101-Ad-hoc-bytemuck-fix-for-https-crbug.com-480176523.patch
deleted file mode 100644
index d5df20dde36..00000000000
--- a/third_party/rust/chromium_crates_io/patches/bytemuck-v1/0101-Ad-hoc-bytemuck-fix-for-https-crbug.com-480176523.patch
+++ /dev/null
@@ -1,40 +0,0 @@
-From 98355e3039f4677b19d1358a207f5ae646b935bb Mon Sep 17 00:00:00 2001
-From: Lukasz Anforowicz <lukasza@chromium.org>
-Date: Fri, 30 Jan 2026 23:30:50 +0000
-Subject: [PATCH] Ad-hoc `bytemuck` fix for https://crbug.com/480176523.
-
-TODO: Remove this patch from `chromium_crates_io/patches` after
-rolling `bytemuck` past the actual fix from
-https://github.com/Lokathor/bytemuck/pull/344
----
- .../rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs        | 1 -
- .../rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs   | 1 -
- 2 files changed, 2 deletions(-)
-
-diff --git a/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs b/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs
-index 846f281d975c7..3aaee72a26e85 100644
---- a/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs
-+++ b/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs
-@@ -151,7 +151,6 @@ impl_unsafe_marker_for_simd!(
- unsafe impl<T, const N: usize> Pod for core::simd::Simd<T, N>
- where
-   T: core::simd::SimdElement + Pod,
--  core::simd::LaneCount<N>: core::simd::SupportedLaneCount,
- {
- }
- 
-diff --git a/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs b/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs
-index d88ba0f6636d0..b0f3b8aca0b08 100644
---- a/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs
-+++ b/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs
-@@ -230,7 +230,6 @@ impl_unsafe_marker_for_simd!(
- unsafe impl<T, const N: usize> Zeroable for core::simd::Simd<T, N>
- where
-   T: core::simd::SimdElement + Zeroable,
--  core::simd::LaneCount<N>: core::simd::SupportedLaneCount,
- {
- }
- 
--- 
-2.53.0.rc1.225.gd81095ad13-goog
-
diff --git a/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs b/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs
index b0397923c71..330f722b341 100644
--- a/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs
+++ b/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/pod.rs
@@ -152,6 +152,7 @@ impl_unsafe_marker_for_simd!(
 unsafe impl<T, const N: usize> Pod for core::simd::Simd<T, N>
 where
   T: core::simd::SimdElement + Pod,
+  core::simd::LaneCount<N>: core::simd::SupportedLaneCount,
 {
 }
 
diff --git a/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs b/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs
index 397dddec99e..186c567fffd 100644
--- a/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs
+++ b/third_party/rust/chromium_crates_io/vendor/bytemuck-v1/src/zeroable.rs
@@ -231,6 +231,7 @@ impl_unsafe_marker_for_simd!(
 unsafe impl<T, const N: usize> Zeroable for core::simd::Simd<T, N>
 where
   T: core::simd::SimdElement + Zeroable,
+  core::simd::LaneCount<N>: core::simd::SupportedLaneCount,
 {
 }
 
